modules/react-native-status/android/src/main/java/im/status/ethereum/module/StatusModule.java

```diff
@@ -236,10 +241,22 @@ private void doStartNode(final String defaultConfig) {
             } catch (Exception e) {
 
             }
+            try {
+                Object clusterConfig = customConfig.get("ClusterConfig");
+                if (clusterConfig != null) {
+                    Log.d(TAG, "ClusterConfig is not null");
+                    jsonConfig.put("ClusterConfig", clusterConfig);
+                }
+            } catch (Exception e) {
+              Log.w(TAG, "Something went wrong parsing cluster config" + e.getMessage());
+            }
+
+
             jsonConfig.put("KeyStoreDir", newKeystoreDir);
 
             config = jsonConfig.toString();
         } catch (JSONException e) {
+            config = Statusgo.GenerateConfig(testnetDataDir, 3);
```

Comment:
I assume that `3` is testnet ID, maybe we could give `3` some name to make it more clear.

Reply:
We have relevant info in `status-im.utils.ethereum.core`

Reply:
Maybe we can reuse one of those methods?